packages/app-treasury/src/index.css

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2017-2019 @polkadot/app-democracy authors & contributors
+/* This software may be modified and distributed under the terms
+/* of the Apache-2.0 license. See the LICENSE file for details. */
+
+.treasury--Item-header {
```

Comment:
Would prefer now having new css files added (I understand why, was easiest at that time since democracy is _very_ similar)